docs/gatsby-node.ts

```diff
@@ -253,6 +259,8 @@ export const createSchemaCustomization: GatsbyNode["createSchemaCustomization"]
       tabCollection: String
       title: String!
       breadcrumbs: [BreadcrumbsPart]
+      storybookLink: String
```

Comment:
instead of extending current types like that with string, do something like that: 

```suggestion
  const typeDefs = [
    `
  type Mdx implements Node {
    frontmatter: MdxFrontmatter!
    fields: MdxFields
  }

  type MdxFrontmatter {
    date: Date @dateformat
    description: String
    redirect_from: [String]
    title: String!
    type: String
  }

  type BreadcrumbsPart {
    name: String!
    url: String!
  }
`,
    schema.buildObjectType({
      name: "MdxFields",
      fields: {
        breadcrumbs: {
          type: "[BreadcrumbsPart]",
        },
        collection: "String!",
        description: "String",
        slug: "String!",
        tabCollection: "String",
        title: "String!",
        hasHeaderLink: {
          type: "Boolean",
          resolve: source => !!source.headerLink,
        },
        hasStorybook: {
          type: "Boolean",
          resolve: source => !source.storybookLink,
        },
      },
    }),
  ];

  actions.createTypes(typeDefs);
```